sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/TypeDeclarationSyntaxExtensions.cs

```diff
@@ -0,0 +1,74 @@
+/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2019 SonarSource SA
```

Comment:
I've just merged the license header update PR. We should now use `Copyright (C) 2015-2020 SonarSource SA` for the 3 new files in this PR